doc/source/Ticket-Creation-and-Definition-of-Ready.rst

```diff
@@ -0,0 +1,103 @@
+Ticket creation and definition of ready
+=======================================
+
+Creating new tickets
+--------------------
+
+User stories
+~~~~~~~~~~~~
+
+When creating new tickets you should follow the template provided:
+
+   As a X I want Y so that Z
+
+   Related issues: #I, #J
+
+   Optional extra information text goes here
+
+   Acceptance criteria: \* A \* B \* C
+
+You should fill in the first sentence, explaining the functionality
+required and who will use it. Make sure you link to any related work,
+especially prerequisites. You can add more detailed descriptions and
+fill in the acceptance criteria to describe what tasks need to be
+completed and what testing might be needed.
+
+A good reference for understanding what a user story is is here:
+http://www.romanpichler.com/blog/10-tips-writing-good-user-stories/
+
+Labelling issues
+----------------
+
+A number of labels are available for helping the categorise the
+available issues. Existing labels are: ### Type \* Administrative: The
```

Comment:
The Type heading is not encoded correctly.

Reply:
Fixed.